lego/api/v1.py

```diff
@@ -151,6 +152,8 @@
     r"meeting-token", MeetingInvitationTokenViewSet, basename="meeting-token"
 )
 router.register(r"meetings", MeetingViewSet, basename="meeting")
+router.register(r"meetingtemplates", MeetingTemplateViewSet, basename="meetingtemplate")
```

Comment:
```suggestion
router.register(r"meetings/templates", MeetingTemplateViewSet, basename="meetingtemplate")
```
I think /templates should be under `meetings. Also, meeting templates are two words